jobs/scheduled/update_category_stats.rb

```diff
@@ -55,16 +56,16 @@ def execute(args = nil)
           category_stats_for_filter = category_stats_for_filter.deep_merge(category_topic_totals)
 
           per_filter_category_stats =
-            per_filter_category_stats.deep_merge({ gft => category_stats_for_filter })
+            per_filter_category_stats.deep_merge({ tag.name => category_stats_for_filter })
         end
 
         c.update!(global_filter_tags_category_stats: per_filter_category_stats)
       end
 
       # Calculate topic totals per GFT
-      filter_tags.each do |gft|
-        filter_tag = GlobalFilter::FilterTag.find_by(name: gft)
-        tag_id = Tag.find_by(name: gft).id
+      filter_tags.each do |tag|
+        filter_tag = GlobalFilter::FilterTag.find_by(name: tag.name)
```

Comment:
I would also probably add a guard here, but this file is not tested enough for me to safely make such a change.